app/vmalert/datasource/vm_prom_api.go

```diff
@@ -161,12 +161,26 @@ func (s *VMStorage) setPrometheusInstantReqParams(r *http.Request, query string,
 		r.URL.Path += "/api/v1/query"
 	}
 	q := r.URL.Query()
-	if s.lookBack > 0 {
-		timestamp = timestamp.Add(-s.lookBack)
+
+	if s.evaluationInterval > 0 {
```

Comment:
I think it is time to move this code to a function. This should simplify the code and make it more testable

Reply:
Good point, added it!